sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/utils-service/application-event-code-during-enrolment-and-completed-utils.service.ts

```diff
@@ -92,20 +89,39 @@ export class ApplicationEventCodeDuringEnrolmentAndCompletedUtilsService {
       : ApplicationEventCode.DISS;
   }
 
+  /**
+   * Checks if there is any full-time disbursement feedback errors.
+   * @param disbursementFeedbackErrors disbursement feedback errors.
+   * @returns true, if there is any full-time disbursement feedback
+   * errors.
+   */
+  private hasFullTimeDisbursementFeedbackErrors(
+    disbursementFeedbackErrors?: Pick<
```

Comment:
I see that the `Pick<DisbursementFeedbackErrors, "errorCode">[]` is used in 2 places, here and at `eventCodeForCompletedApplicationWithSentDisbursement()` we can create a type to which can be used in both places.